pkg/apis/policy/validation/validation.go

```diff
@@ -194,6 +203,28 @@ func validatePSPAllowedFlexVolumes(fldPath *field.Path, flexVolumes []policy.All
 	return allErrs
 }
 
+func validatePSPAllowedCSIDrivers(fldPath *field.Path, csiDrivers []policy.AllowedCSIDriver) field.ErrorList {
+
+	allErrs := field.ErrorList{}
+	if len(csiDrivers) > 0 {
+		for idx, csiDriver := range csiDrivers {
+			if len(csiDriver.Name) == 0 {
```

Comment:
same here regarding shared validation methods

Reply:
See response in newer PR here - https://github.com/kubernetes/kubernetes/pull/74086#issuecomment-464188838